test/extended/clusterup.sh

```diff
@@ -158,6 +158,21 @@ function os::test::extended::clusterup::hostdirs () {
     BASE_DIR="${base_dir}" os::test::extended::clusterup::internal::hostdirs
 }
 
+# Tests bringing up the service catalog and provisioning a template
+function os::test::extended::clusterup::service_catalog() {
+    os::cmd::expect_success "oc cluster up --service-catalog --version=latest"
+    os::test::extended::clusterup::verify_router_and_registry
+    os::test::extended::clusterup::verify_image_streams
+    os::cmd::expect_success "oc login -u system:admin"
+    os::cmd::expect_success "oc adm policy add-cluster-role-to-group system:openshift:templateservicebroker-client system:unauthenticated system:authenticated"
+    #oc::cmd::expect_success "oc adm policy add-role-to-user view developer -n openshift"
```

Comment:
quick summary of exchange @bparees and I had on IRC: ... odd rc=127 is coming back on the `oc adm policy add-role-to-user ..` or `oc polic ..` invocations from the clusterup.sh framework.

Implies the oc command does not exist (though obviously it is leveraged elsewhere).

Ultimately orthogonal to this card ... I'm not pursuing further at this time.